superwall/src/main/java/com/superwall/sdk/config/ConfigManager.kt

```diff
@@ -105,9 +109,8 @@ open class ConfigManager(
                 }
 
             // Await results from both operations
-            val config = configDeferred.await()
-            geoDeferred.await()
-
+            val (result, _) = listOf(configDeferred, geoDeferred).awaitAll()
```

Comment:
Is there maybe a reason why the calls were sequential? If not, I'd use this so they are parallel.

Reply:
There's no reason, it can be parallel - it is on iOS

Reply:
Yeah just noticed, awesome 👍